apps/web/src/app/lootrun/_component/loot-item.tsx

```
@@ -0,0 +1,206 @@
+    }
+  }
+
+  // Special case for types not in validIcons but in WynnIcon logic
+  if (item.itemType === "TomeItem") return <ItemTypeIcon type="tome" size={32} />
+
+  // Specific fallbacks for common types
+  switch (item.itemType) {
+    case "GearItem":
+      return <Sword className="h-5 w-5" />
+    case "AspectItem":
+      const aspectClass = item.subtype?.split("Aspect")[0].toLowerCase()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)